task.css

```diff
@@ -0,0 +1,83 @@
+
+.container {
+    font-family:'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-size: small;
+    display: flex;
+    flex-direction: row;
+    padding:0px 0px 0px 0px;

```

Comment:
`padding: 0px` sets all the 4 paddings to 0.


Anyhow, this is the default padding, no need to explicitly set it.